capstone/src/main/webapp/script.js

```diff
@@ -118,8 +119,9 @@ function precinctControl(controlDiv, map) {
   precinctLayer.loadGeoJson('neighborhoods.json');
   precinctLayer.setStyle({visible: false});
   precinctLayer.addListener('click', function(event) {
-    document.getElementById('sentiment-pie-chart').textContent =
-     event.feature.getProperty('station');
+    precinct = event.feature.getProperty('station');
+    associationUpdateDisplay();
```

Comment:
I accidentally didn't get rid of this, if you make changes maybe just remove the association precinct stuff while you are at it (it doesn't affect functionality its just dead code).